docs/tutorial/backup-restore.md

```diff
@@ -0,0 +1,106 @@
+# MyDuck Server Backup and Restore Guide
+
+## Introduction
+
+MyDuck Server leverages DuckDB's high-performance analytical capabilities to manage and analyze your data efficiently. This guide provides comprehensive instructions on how to perform backups and restores using the enhanced mechanisms introduced in MyDuck Server, ensuring quick and reliable data management.
+
+## Backup
+
+### Motivation
+
+DuckDB natively supports [`EXPORT/IMPORT DATABASE TO/FROM OBJECT STORAGE`](https://duckdb.org/docs/sql/statements/export), where the schema is stored in `schema.sql` and data in `.csv` or `.parquet` files. However, the restore process requires executing the SQL from `schema.sql` and then importing all corresponding data files, which can be time-consuming.
+
+To optimize backup and restore times, MyDuck Server now allows you to backup the entire database into a single file (`mysql.db`). This approach simplifies the backup and restore process, enabling users to download and directly attach the backup file during a cold start, significantly reducing restoration time.
+
+**Note:** During the backup process, the server switches to read-only mode, rejecting all DML (Data Manipulation Language) and DDL (Data Definition Language) operations. Once the backup completes, the server automatically reverts to read-write mode.
+
+### Backup Syntax
+
+```sql
+BACKUP DATABASE my_database TO '<uri>'
+  ENDPOINT = '<endpoint>'
+  ACCESS_KEY_ID = '<access_key>'
+  SECRET_ACCESS_KEY = '<secret_key>'
+```
+
+**Example:**
+
+```sql
+BACKUP DATABASE my_database TO 's3://my_bucket/my_database/'
+  ENDPOINT = 's3.cn-northwest-1.amazonaws.com.cn'
+  ACCESS_KEY_ID = 'xxxxxxxxxxxxx'
+  SECRET_ACCESS_KEY = 'xxxxxxxxxxxx'
+```
+
+**Notes:**
+- `s3` refers to AWS S3.
+- `s3c` refers to other S3-compatible object storage services, such as MinIO or Aliyun OSS.
```

Comment:
`Aliyun` -> `Alibaba Cloud OSS`

It looks better to add a hyperlink for each of them.